addons/binding/org.openhab.binding.yamahareceiver/ESH-INF/thing/thing-types.xml

```diff
@@ -64,7 +71,10 @@
 		<description>Control the zone</description>
 		<channels>
 			<channel id="power" typeId="power" />
+			<!--
+            We will provide an input channel dynamically. 
```

Comment:
It might have been a comment about dynamic channel types, while without dynamic channels many cases relying on runtime device configuration would be impossible to achieve.